.github/workflows/push-release.yml

```diff
@@ -382,17 +382,25 @@ jobs:
           password: ${{ env.DOCKER_PASSWORD }}
       - name: Create manifest list and push
         run: |
-          docker buildx imagetools create -t ${{ env.IMAGE_AGENT }}:${{ env.TAG }} \
-            ${{ env.IMAGE_AGENT }}:${{ env.TAG }}-amd64 \
-            ${{ env.IMAGE_AGENT }}:${{ env.TAG }}-arm64 \
+          # docker manifest is used with windows images to maintain os.version in the manifest
+          docker manifest create ${{ env.IMAGE_AGENT }}:${{ env.TAG }} \
             ${{ env.IMAGE_AGENT }}:${{ env.TAG }}-windows-2019 \
             ${{ env.IMAGE_AGENT }}:${{ env.TAG }}-windows-2022
+
+          docker manifest push ${{ env.IMAGE_AGENT }}:${{ env.TAG }}
+
+          docker buildx imagetools create -t ${{ env.IMAGE_AGENT }}:${{ env.TAG }} \
+            --append ${{ env.IMAGE_AGENT }}:${{ env.TAG }}-amd64 \
+            --append ${{ env.IMAGE_AGENT }}:${{ env.TAG }}-arm64 \
```

Comment:
We need to remove the trailing slash to avoid https://github.com/rancher/rancher/issues/46639